### ISSUE TITLE
Cashier webhook artisan command uses incorrect default path

### ISSUE THREAD

**Issue by markkimsal** [laravel/cashier-stripe#1292]
- Cashier Version: 13.6
- Laravel Version: 8.77.1
- PHP Version: 7.3.30

### Description:
When following the upgrade instructions to create a new webhook `php artisan cashier:webhook --disabled` I get an error

```
   Symfony\Component\Routing\Exception\RouteNotFoundException 
                                                                                   
  Route [cashier.webhook] not defined.                                                                                                                                

  at vendor/laravel/framework/src/Illuminate/Routing/UrlGenerator.php:444
    440▕         if (! is_null($route = $this->routes->getByName($name))) {                                                                                           
    441▕             return $this->toRoute($route, $parameters, $absolute);
    442▕         }                                                                 
    443▕                                                                                                                                                                ➜ 444▕         throw new RouteNotFoundException("Route [{$name}] not defined.");
    445▕     }
    446▕                                                                           
    447▕     /**                                                                                                                                                      
    448▕      * Get the URL for a given route instance.
                                                                                   
  1   vendor/laravel/framework/src/Illuminate/Foundation/helpers.php:739                                                                                              
      Illuminate\Routing\UrlGenerator::route("cashier.webhook", [])
                                                                                   
  2   vendor/laravel/cashier/src/Console/WebhookCommand.php:43                                                                                                        
      route("cashier.webhook")
                                         
```
### Steps To Reproduce:

run `php artisan cashier:webhook`

### Solution
You must override the default cashier config path of `stripe` to `cashier`.  Editing your .env to have
```
CASHIER_PATH=cashier
```
will allow the artisan command to run.

The default value of laravel-cashier/config/cashier.php should have "cashier" as the path instead of "stripe", or the webhook command should not rely on 

```
'url' => $this->option('url') ?? route('cashier.webhook'),
```
since this route doesn't exist by default.

**Comment by driesvints** [comment]
This route always exists when you have Cashier Stripe installed unless you're overwriting Cashier's routes. Did you overwrite the route in your own route file and maybe forgot to add the route name?

Also please note that this command is relying on a publicly accessible web hook. It was more intended to be used in production for first setting up your web hook after launching your app. For local dev, we recommend using the Stripe CLI tool.

**Comment by markkimsal** [comment]
I have overridden "stripe/webhook", but not "cashier/webhook".  The cashier prefixed route does not exist as far as I can tell in a vanilla installation without customizing the .env.

https://github.com/laravel/cashier-stripe/blob/dcdd389f245a338854ca989cef300557d21f5025/src/CashierServiceProvider.php#L91

```php
    protected function registerRoutes()
    {
        if (Cashier::$registersRoutes) {
            Route::group([
                'prefix' => config('cashier.path'),
                'namespace' => 'Laravel\Cashier\Http\Controllers',
                'as' => 'cashier.',
            ], function () {
                $this->loadRoutesFrom(__DIR__.'/../routes/web.php');
            });
        }
    }
 ```
 
 https://github.com/laravel/cashier-stripe/blob/dcdd389f245a338854ca989cef300557d21f5025/config/cashier.php#L31
 
 ```php
    'path' => env('CASHIER_PATH', 'stripe'),
```
The prefix of the Route comes from `config('cashier.path')` which defaults to `stripe`, not `cashier` as expected by the artisan command.

**Comment by markkimsal** [comment]
I see now.  I did not have the `->name('cashier.webhook')` in my customized `stripe/webhook` route.  Therefore, when dealing with the default path of `stripe` the system noticed the route was already created.

I was confusing "prefix" and "as" in the Route with an already overridden route that did not define a name.  That's why it worked when I changed the prefix, because now the fresh routes `cashier/webhook` was not already defined, and it was created with the proper alias/name.
